.circleci/config.yml

```diff
@@ -19,9 +19,16 @@ jobs:
       - checkout
       - run: make unit-test
       - run: bash <(curl -s https://codecov.io/bash)
+  bddtests:
+    docker:
+      - image: circleci/golang:1.12
```

Comment:
@talwinder50 shouldn't the bddtests be implemented in a custom image?

Reply:
@llorllale Is the circleci image missing something?

Reply:
@troyronda @talwinder50 nevermind my previous comment